api/nakadi-event-bus-api.yaml

```diff
@@ -1503,11 +1539,39 @@ definitions:
         type: string
         default: 'random'
 
+      compatibility_mode:
+        description: |
+          Compatibility mode provides a mean for event owners to evolve their schema, given changes respect the
+          semantics defined by this field.
+
+          It's designed to be flexible enough so that producers can evolve their schemas while not
+          inadvertently breaking existent consumers.
+
+          Once defined, the compatibility mode is fixed, since otherwise it would break a predefined contract,
+          declared by the producer.
+
+          List of compatibility modes:
+
+          - 'deprecated': Schema changes are not allowed. This is used only by legacy event types. This option is not
```

Comment:
i'm wondering if there's a way to not have this option; it's very specific to the state of a deployment. Would it possible to default existing schemas to none and allow them to stay on none or move up to compatible (if the schema in place meets the compatibility rules). 

Alternatively if it's being kept it could be renamed to "fixed" instead of "deprecated".